SwiftyTesseract/SwiftyTesseract/SwiftyTesseract.swift

```diff
@@ -225,18 +225,17 @@ public class SwiftyTesseract {
   ///   - images: Array of UIImages to perform OCR on
   ///   - path: Path where the PDF document should be saved.
   /// - Throws: SwiftyTesseractError
-  public func createPDF(from images: [UIImage], at path: URL) throws {
-
+  @discardableResult
+  public func createPDF(from images: [UIImage], at path: URL) throws -> URL {
     let _ = semaphore.wait(timeout: .distantFuture)
     defer {
       semaphore.signal()
     }
 
-    // create unique file path
     if path.pathExtension.lowercased() == "pdf" {
-      try processPDF(images: images, at: path.deletingPathExtension())
+      return try processPDF(images: images, at: path.deletingPathExtension())
```

Comment:
I just realized this, but since `validate(_:)` is doing this work for us, we don't even need to check for the pdf path extension defensively, we can just past it down and completely remove the conditional statements here.